.prettierrc.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @type {import('prettier').Options}
+ */
+module.exports = {
+  printWidth: 90,
+  semi: true,
+  singleQuote: false,
+  trailingComma: "all",
+  bracketSpacing: true,
+  bracketSameLine: true,
+  importOrder: ["^@chainpatrol/(.*)$", "~/(.*)$", "~(.*)$", "^[./]"],
+  importOrderSeparation: true,
+  importOrderSortSpecifiers: true,
+  importOrderCaseInsensitive: false,
+  plugins: [require.resolve("@trivago/prettier-plugin-sort-imports")],
+};
```

Comment:
Is this copied from the `chainpatrol-web` repo?

Reply:
Yeah, it's the same config copied. I was thinking having the same style configurations be consistent would be better so that anyone contributing to the bot could be aligned.